packages/component-base/src/styles/style-props.js

```diff
@@ -94,8 +102,8 @@ addGlobalThemeStyles(
           --vaadin-border-color: CanvasText;
           --vaadin-border-color-strong: CanvasText;
```

Comment:
Let’s fix this at the same time. A leftover from a previous renaming.
```suggestion
          --vaadin-border-color-secondary: CanvasText;
```

Reply:
Done.